public/app/plugins/datasource/alertmanager/types.ts

```diff
@@ -162,6 +162,13 @@ export enum MatcherOperator {
   notRegex = '!~',
 }
 
+export enum MatcherOperatorOptions {
```

Comment:
this enum kinda duplicates `MatcherOperator`. I'd leave only one of these to avoid confusion